nx/exercises/exercises-1-20.livemd

```diff
@@ -0,0 +1,174 @@
+# Exercises: 1-20
+
+```elixir
+Mix.install([{:nx, "~> 0.6"}])
+```
+
+## Introduction
+
+Inspired by the Python notebook _100 Numpy Exercises_:
+
+https://www.kaggle.com/code/utsav15/100-numpy-exercises/notebook
+
+## 1-10
+
+**1. Install Nx in a Livebook (★☆☆)**
+
+```elixir
+Mix.install([{:nx, "~> 0.6"}])
+```
+
+**2. Create a 1-D tensor of 10 zeros (★☆☆)**
+
+```elixir
+Nx.broadcast(0, {10})
+```
+
+**3. How to find the number of elements in a tensor (★☆☆)**
+
+```elixir
+Nx.tensor([[1, 2, 3], [4, 5, 6]])
+|> Nx.size()
+```
+
+**4. How to find the memory size of any tensor (★☆☆)**
+
+```elixir
+Nx.tensor([[1, 2, 3], [4, 5, 6]])
+|> Nx.byte_size()
+```
+
+**5. How to get the documentation of the `add/2` function from the command line? (★☆☆)**
+
+```elixir
+# I don't know if this is possible.
+# iex> h Nx.add
```

Comment:
This should be possible. In livebook you can use `import IEx.Helpers` to get access to the `h` helper

Reply:
[4be316e](https://github.com/elixir-nx/nx/pull/1363/commits/4be316ed841dec5479a7830fa57037ca9f6fe894)